web_frontend/src/App.tsx

```diff
@@ -1,14 +1,12 @@
 import { useEffect } from "react";
 import "./index.css";
-import { useTranscription } from "./hooks/useTranscription";
-import { useUiUpdateBackendPoll } from "./hooks/useUiUpdateBackendPoll";
 import { generateRandomUserId, setUserIdAndDeviceId } from "./utils/utils";
 import Cookies from "js-cookie";
-import MainLayout from "./layouts/MainLayout";
+import StudyLayout from "./layouts/StudyLayout";
+import { useTrackTabChange } from "./hooks/useTrackTabChange";
 
 export default function App() {
-  useTranscription();
-  useUiUpdateBackendPoll();
+  useTrackTabChange();
 
```

Comment:
Could you use a variable to disable the backend bits instead of hardcoding the changes please?

Reply:
That makes sense, just pushed a change for this.

Reply:
This is fine where it's easy, but in general don't lose any time making this backwards compatible - we will make some quick iterations on the frontend during first pilots of the study. Our data and learning at the end will influence what we use/don't use, and we can properly integrate those developments in after the study.